zerver/tests/test_messages.py

```diff
@@ -1271,16 +1271,6 @@ def test_send_message_irc_mirror(self) -> None:
                                    "subject": "from irc",
                                    "to": "IRCLand"},
                                   **self.api_auth(email))
-        self.assert_json_error(result, "IRC stream names must start with #")
```

Comment:
Is this test still testing an irc_mirror pathway? If it's just essentially sending a regular message, we should remove the test. (You can use the --coverage option to test-backend.py to help figure this out; feel free to post in #development help if that doesn't make sense.)

Reply:
@rishig All the functions (in `zerver/views/messages.py`) related to irc_mirrors are running as shown by the test coverage of this function. So I think we should keep this in

Reply:
ok, great